data/xapleaux/xapleaux missions.txt

```diff
@@ -0,0 +1,192 @@
+
+			`	Looking at the lead alien of the three approaching, you're fairly sure your own arms wouldn't be quite able to reach entirely around it, but its uppermost arms certainly could.`
+
+			choice
+				`	(Raise your hands peaceably.)`
+					goto peace
+				`	(Hold out one hand before you to indicate a halt and draw your gun.)`
+					goto caution
+				`	(Draw and point your gun directly at these monstrosities.)`
+			`	There is an electric thrill of activity that ripples through them the moment you point your gun. The three approaching you stop and in the periphery you sense a greatly increased level of shuffling and shifting 'foliage'.`
+			choice
+				`	(Shoot first.)`
+					goto shoot
+				`	(Point the gun at the movement in your periphery.)`
+				`	(Lower the gun slightly and look around.)`
+					goto suspicion
+			`	You swing your gun towards the greatest point of activity in your periphery and are treated to the sight of an alien pointing a weapon that requires four of its arms to handle. The barrel on the weapon is nearly as wide around as the alien carrying it, and your last thought becomes something along the lines of, "Is that a cannon?!" before a brilliant white flash puts an end to you.`
+				die
+			label shoot
+			`	Your weapon goes off and you watch the shot impact and injure, but not severely, the lead alien approaching you. You have barely an instant to register the whir of something charging up in your periphery before white-hot pain makes further considerations redundant.`
+
+				die
+			label suspicion
+			`	Pointing your weapon back at the ground you take a look around at the shifting crowd keeping its distance. You get the sudden sense that you spared yourself a sudden demise as you catch sight of an alien who had emerged to stand in your periphery, pointing a weapon at you that requires four of its arms to handle. The barrel on the weapon is nearly as wide around as the alien carrying it, and you can't help but think it looks alarmingly like a cannon, possibly one suitable for mounting on a ship.`
+
+			`	The lead alien approaching makes a sharp clicking noise and steps forward once, drawing your attention back to it.`
+			label caution
+			`	It holds its two upper arms up and out, seemingly in a gesture not dissimilar to the human gesture indicating peace. Perhaps simply indicating empty appendages is a necessary evolutional feature for a society. Its middle arms though point at the gun, then at the satchel at its side, then at you, then at your holster, and then traces a line on the ground from it to you as if to indicate it would approach.`
```

Comment:
```suggestion
			`	It holds its two upper arms up and out, seemingly in a gesture not dissimilar to the human gesture indicating peace. Perhaps simply indicating empty appendages is a necessary evolutional feature for a society. However, its middle arms point at the gun, then at the satchel at its side, then at you, then at your holster, and then traces a line on the ground from it to you as if to indicate it would approach.`
```